guide/src/contributing/code-formatting.md

```diff
@@ -16,5 +16,25 @@ Once that is taken care of, you can (re)format all code by running this command
 from the root of the repository:
 
 ```
-$ cargo +nightly fmt --all
+$ cargo fmt --all
+```
+
+# Linting
```

Comment:
Linting setup help here, this more or less follows the same flow as the `rustfmt` help above it.